tap_square/schemas/items.json

```diff
@@ -1,45 +1,229 @@
 {
-  "type": [
-    "null",
-    "object"
-  ],
   "properties": {
     "id": {
       "type": [
         "null",
         "string"
       ]
     },
-    "is_deleted": {
+    "updated_at": {
       "type": [
         "null",
-        "boolean"
+        "string"
       ]
```

Comment:
Assuming this will need to be a datetime? Not sure when we feel comfortable combing over the data types. That's something I did last week for adroll.

Reply:
I'm sure there could be more, just saw that one jump out at me. Going to ignore others as I review.

Reply:
Interesting, this seems like a bug in the infer-schema tool to me. I think that should be a datetime.